News/Services/UserService.cs

```diff
@@ -109,7 +109,7 @@ CancellationToken cancellationToken
             throw new Exception("User not found");
         }
 
-        var password = await _passwordRepo.GetByIdAsync(user.PasswordId, cancellationToken);
+        var password = await _passwordRepo.GetByIdAsync(user.PasswordId.Value, cancellationToken);
```

Comment:
why .PasswordId.Value?

Reply:
without value, an error occurs
![image](https://github.com/user-attachments/assets/67c2508b-469b-47b6-8a8d-6f473201543a)

Reply:
@MaxGudinovich take a look please

Reply:
I guess because of **that**:

```
 public async Task<T> GetByIdAsync(Guid id, CancellationToken cancellationToken)
    {
        return await _dbSet.FindAsync(new object[] { id }, cancellationToken);
    }
    
```
**To avoid errors, you can:**

Explicitly convert id from Guid? to Guid before calling FindAsync, for example: FindAsync(new object[] { id.Value }, cancellationToken);—ensuring that id is not null beforehand.

Alternatively, use FirstOrDefaultAsync, as in the second approach, if additional flexibility is needed.

Reply:
I fixed it, in reality, it was very simple, as always—just one wrong character, and nothing works 

We’ll use FindAsync(id, ...) because we’re only looking for the id without any additional conditions, plus FindAsync is faster due to caching

Reply:
@IKonrad98 
Btw Guid? inside UserEntity it is not a misstake.